contracts/VotingUtils.sol

```diff
@@ -118,7 +119,7 @@ contract VotingUtils is EternalStorage, VotingBase {
     if (getStartBlock(_id) > block.number) return false;
     if (getIsFinalized(_id)) return false;
 
-    return block.number > getEndBlock(_id);
+    return block.number >= getEndBlock(_id);
```

Comment:
This fixes the bug were voting takes one cycle longer to finish than it should

Reply:
that's cool!